README.md

```diff
@@ -41,112 +43,120 @@ conctr <- Conctr(APP_ID, API_KEY, MODEL);
 
 ## Agent Class Methods
 
-### setDeviceId(*[,deviceId]*)
+### setDeviceId(*[deviceId]*)
 
-The *setDeviceId()* allows you the set the unique identifier that will be used by Conctr to identify the current device. 
+The *setDeviceId()* method allows you the set the unique identifier that will be used by Conctr to identify the current device. 
 
-**NOTE: Changing the device id after data has already been set previously will create a new device in Conctr. There will be no link between any data from this newly created device and the device data linked to the previous device id (if any).**
+**Note** Changing the device ID after data has already been set previously will create a new device in Conctr. There will be no link between any data from this newly created device and the device data linked to the previous device ID.
 
 | Key | Data Type | Required | Default Value | Description |
-| ----| --------------- | --------- | ----------- |----------- |
-| *deviceId* | String | No | `imp.configparams.deviceid` | Custom unique identifier that Conctr should store data against for this device. |
+| --- | --------- | -------- | ------------- | ----------- |
+| *deviceId* | String | No | `imp.configparams.deviceid` | Custom unique identifier that Conctr should store data against for this device |
 
-##### Example
+#### Example
 
 ```squirrel
-const CUSTOM_DEVICE_ID="device-1";
+const CUSTOM_DEVICE_ID = "device-1";
 
 conctr.setDeviceId(CUSTOM_DEVICE_ID);
 ```
 
+
 ### sendData(*payload[, callback]*)
 
-The *sendData()* method sends a data payload to Conctr via the data ingeston endpoint. It is called by the data event listener when the device sends data using the device Conctr class. It can also be used directly to send data to Conctr via the agent alone.
+The *sendData()* method sends a data payload to Conctr via the data ingeston endpoint. 
 
-| Key | Data type | Required | Description |
-| ----| --------------- | --------- | ----------- |
-| *payload* | Table/Array of Tables | Yes | A table or array containing the data to be sent to Conctr. The keys of each table in the data should correspond to fields from the model and/or Conctr metadata fields and should be of the type specified in the model.|
-| *callback* | Function | No | Function to be called on response from Conctr. function should take two arguements, error and response. When no error occurred the first arguement will be null.|
+| Key | Data Type | Required | Description |
+| --- | --------- | -------- | ----------- |
+| *payload* | Table/Array | Yes | A table or array of tables containing the data to be sent to Conctr. The keys in the table should correspond to fields from the model and the keys should be of type specified in the model. |
+| *callback* | Function | No | Function to be called on response from Conctr. The function should take two arguements, *error* and *response*. When no error occurred, the first arguement will be null. |
```

Comment:
should probably have a callback table.
See: https://github.com/mysticpants/Wiznet_5500
e.g
This method triggers the supplied callback function when data is received. The callback takes the following parameters:
Key            | Data Type|Description 
----------  | ---------- | ----------
error           | String .     | An error message
response   |